src/routes/routing.route.ts

```diff
@@ -27,6 +27,7 @@ router.get('/popular', popularRoutes);
  *     summary: some summary
  *     description: some description
 */
+router.get('/getRoutes', getRoutes);
```

Comment:
The endpoint for getting routes should be just `/routes`, since GET is already the type of HTTP request